tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -308,17 +301,10 @@ where
                 (FmtLevel::new(meta.level()), FmtCtx::new(&ctx))
             }
         };
-        write!(
-            writer,
-            "{} {}{}: ",
-            fmt_level,
-            fmt_ctx,
-            if self.display_target {
-                meta.target()
-            } else {
-                ""
-            }
-        )?;
+        write!(writer, "{} {}", fmt_level, fmt_ctx,)?;
```

Comment:
nit: unnecessary trailing comma here?
```suggestion
        write!(writer, "{} {}", fmt_level, fmt_ctx)?;
```